azure-pipelines-1.yml

```
@@ -0,0 +1,19 @@
+# Starter pipeline
+# Start with a minimal pipeline that you can customize to build and deploy your code.
+# Add steps that build, run tests, deploy, and more:
+# https://aka.ms/yaml
+
+trigger:
+- master
```

YAMLlint (1.35.1)

[error] wrong indentation: expected at least 1

(indentation)